assets/scripts/main.js

```diff
@@ -0,0 +1,62 @@
+let books = [];
```

Comment:
- [ ] Please Make sure that data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md). I can see that you preserved the data correctly but you are not getting the data back from the localstorage. The list of books disappears when the page reloads. Kindly fetch from localstorage and display on reload of the page.